var/spack/repos/builtin/packages/lbann/package.py

```diff
@@ -316,39 +317,7 @@ class Lbann(CMakePackage, CudaPackage, ROCmPackage):
     depends_on("zstr")
 
     generator("ninja")
-
-    @property
-    def common_config_args(self):
-        spec = self.spec
-        # Environment variables
-        cppflags = []
-        cppflags.append("-DLBANN_SET_EL_RNG")
-        cppflags.append("-std=c++17")
-        args = []
-        args.extend(["-DCMAKE_CXX_FLAGS=%s" % " ".join(cppflags), "-DLBANN_VERSION=spack"])
-
-        if "+numpy" in spec:
-            args.append("-DCNPY_DIR={0}".format(spec["cnpy"].prefix))
-
-        # Use lld high performance linker
-        if "+lld" in spec:
-            args.extend(
-                [
-                    "-DCMAKE_EXE_LINKER_FLAGS=-fuse-ld=lld",
-                    "-DCMAKE_SHARED_LINKER_FLAGS=-fuse-ld=lld",
-                ]
-            )
-
-        # Use gold high performance linker
-        if "+gold" in spec:
-            args.extend(
-                [
-                    "-DCMAKE_EXE_LINKER_FLAGS=-fuse-ld=gold",
-                    "-DCMAKE_SHARED_LINKER_FLAGS=-fuse-ld=gold",
-                ]
-            )
-
-        return args
+    depends_on("ninja", type="build")
```

Comment:
This is inherited from `CMakePackage` (a superclass of `CachedCMakePackage`) so you should not have to define it here.

Reply:
Done.